modules/table/src/integrationTest/java/org/apache/ignite/internal/table/distributed/disaster/AbstractHighAvailablePartitionsRecoveryTest.java

```diff
@@ -132,10 +142,71 @@ private void waitAndAssertStableAssignmentsOfPartitionEqualTo(
                 .collect(Collectors.toUnmodifiableSet()));
     }
 
+    final void waitAndAssertEmptyRebalanceKeysAndStableAssignmentsOfPartitionEqualTo(
```

Comment:
The name of the method is a bit ambiguous, it could be understood that stable is also must be null. Please improve naming and a javadoc explaining what do we assert in this method

Reply:
Done